commands/env.cmd

```diff
@@ -29,6 +29,12 @@ if [[ ${WARDEN_ENV_TYPE} != local ]]; then
     WARDEN_REDIS=${WARDEN_REDIS:-1}
 fi
 
+if [[ ${WARDEN_ENV_TYPE} == "magento1" && -f "${WARDEN_ENV_PATH}/.modman/.basedir" ]]; then
+  export WARDEN_MODMAN_ROOT='/'`cat "${WARDEN_ENV_PATH}/.modman/.basedir"`
+else
+  export WARDEN_MODMAN_ROOT='/'
```

Comment:
I think the name is weird. Yes, we use it for a `modman`-specific fix, but it is used in all cases.

1. Maybe we can directly set it to `NGINX_PUBLIC`?
2. If not, maybe we use a generic name like `NGINX_PUB_DIR`?

Reply:
Directly is not possible, at least I did not find a way.
So I find this naming already better, you're right